OloEngine/src/OloEngine/Scene/SpatialAcceleration.cpp

```
@@ -0,0 +1,270 @@
+    SceneSpatialIndex::SceneSpatialIndex(f32 cellSize)
+    {
+        if (cellSize <= 0.0f)
+        {
+            OLO_CORE_WARN("SceneSpatialIndex: invalid cellSize {}, clamping to 1.0f", cellSize);
+            cellSize = 1.0f;
+        }
+        m_CellSize = cellSize;
+        m_InvCellSize = 1.0f / cellSize;
+    }
+
+    void SceneSpatialIndex::Clear()
```

Cppcheck (2.21.0)

[style] The function 'Clear' is never used.

(unusedFunction)